### PR TITLE
Consolidated state level additions variables across all state models

### LINKED ISSUE #3650: Consolidate state level additions variables across all state models
Currently we use the following iterations:

`[state_code]_additions`
`[state_code]_income_additions`
`[state_code]_agi_additions`

we should consolidate all to 
`[state_code]_additions`

### PR DESCRIPTION
**Pull request by masterismail**
Fixes #3650

### CODE REVIEW

File: policyengine_us/variables/gov/states/ct/tax/income/ct_additions.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ct/tax/income/ct_additions.py:1]: this seems to be a duplicate file - we already have a `ct_additions` file - please delete one of them

File: policyengine_us/variables/gov/states/me/tax/income/adjusted_gross_income/me_additions.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class me_agi_additions(Variable):
+class me_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "ME AGI additions"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/adjusted_gross_income/me_additions.py:7]:
```suggestion
    label = "Maine AGI additions"
```
While we are in here

File: policyengine_us/variables/gov/states/ne/tax/income/ne_additions.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class ne_agi_additions(Variable):
+class ne_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "NE AGI additions to federal AGI"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ne/tax/income/ne_additions.py:7]:
```suggestion
    label = "Nebraska AGI additions to federal AGI"
```

File: policyengine_us/variables/gov/states/ny/tax/income/adjusted_gross_income/ny_additions.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class ny_agi_additions(Variable):
+class ny_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "NY AGI additions"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ny/tax/income/adjusted_gross_income/ny_additions.py:7]:
```suggestion
    label = "New York AGI additions"
```

File: policyengine_us/variables/gov/states/or/tax/income/or_additions.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class or_income_additions(Variable):
+class or_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "OR income additions"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/or/tax/income/or_additions.py:7]:
```suggestion
    label = "Oregon income additions"
```

File: policyengine_us/variables/gov/states/wi/tax/income/additions/wi_additions.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 from policyengine_us.model_api import *
 
 
-class wi_income_additions(Variable):
+class wi_additions(Variable):
     value_type = float
     entity = TaxUnit
     label = "WI additions to federal adjusted gross income"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/wi/tax/income/additions/wi_additions.py:7]:
```suggestion
    label = "Wisconsin additions to federal adjusted gross income"
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: changelog_entry.yaml [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
[the remark this answers is not in the document]
Comment (masterismail) [reply changelog_entry.yaml]: done

File: policyengine_us/variables/gov/states/ct/tax/income/ct_additions.py
Comment (masterismail) [reply policyengine_us/variables/gov/states/ct/tax/income/ct_additions.py:1]: done

**Comment by PavelMakarchuk** [comment]
@masterismail some tests still seem to fail, could take a look at the usage of:
`ut_additions_to_income` , `nj_agi_additions` , and `me_agi_additions` across the model and adjust where needed

**Comment by masterismail** [comment]
> @masterismail some tests still seem to fail, could take a look at the usage of: `ut_additions_to_income` , `nj_agi_additions` , and `me_agi_additions` across the model and adjust where needed

fixed them ! apologies for the oversight

**Comment by PavelMakarchuk** [comment]
> Details

@masterismail No worries, it takes some time to dig through the whole code base and get familiar with it. I think the last file that needs should be `wi_income_additions`  , my bad I missed it initially.

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**